### DISCUSSION TITLE
Dear readfish developers

### DISCUSSION THREAD

**Discussion by IguodalaYan** [LooseLab/readfish#254] [Q&A]
I would like to know if miniknow integrates the readfish. Is the adaptive sequencing mode in miniknow the same as readfish?
We have a library with target genes and other gene sequences that are approximately 700bp long. We feel that the mode in miniknow may not effectively distinguish them. We would like to set the break_reads_after_seconds to 0.4 or even lower, so we installed readfish. If it is possible to achieve the same functionality within miniknow, could you please tell us how to do it? Thank you very much.

**Comment by mattloose** [comment]
Sorry for the delay in answering this.

We can't really advise on making changes to adaptive sampling in MinKNOW itself but you can can set break_reads_after_seconds to 0.4 seconds and MinKNOWs adaptive sampling should automatically adjust.

However you should not go any lower than 0.4 seconds as MinKNOW will not function correctly. We cannot guarantee that MInKNOW will function correctly using any changes you may make.

If I understand your question correctly, your targets are small - but it also seems that you library may be short?

Adaptive sampling isn't best at looking for very short targets but if you know the sequence around the target you can simply make larger targets. 

If your library is only short molecules (i.e around 700 bases) we really do not advise using adaptive sampling at all. 

I hope this helps a littl.
